szcc

```diff
@@ -30,14 +30,31 @@ def arg(flag, values):
             cmd += f" -{flag}{val}"
     return cmd
 
-def findexec(filename,path,failonmissing=False):
+def modifypath(path,remove=None,prepend=[],append=[]):
+    """ Modify PATH-style variable """
+    tpath = path.split(':')
+    opath = list()
+    opath.append(prepend)
+    for x in tpath:
+        if not (remove and x != remove):
```

Comment:
I believe `x != remove` should be negated.

Or perhaps you meant `not remove or x != remove`?

Reply:
Good catch 👍